src/lib/redux/inventoryDataSlice.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { InventoryRecord, InventoryUpdateRecord, ProductRecord, PurchaseRequestRecord } from '../airtable/interface';
+
+interface inventoryDataSliceState {
+  isLoading: boolean;
+  products: ProductRecord[];
+  siteInventory: InventoryRecord[];
+  purchaseRequests: PurchaseRequestRecord[];
+  inventoryUpdates: InventoryUpdateRecord[];
+}
+
+const initialState: inventoryDataSliceState = {
+  isLoading: false,
```

Comment:
`isLoading` was initially necessary because we sent API request and had to wait for it to be fulfilled. I think this isn't necessary anymore because we no longer send API request (we just use `siteData`'s)